buildSrc/src/main/java/com/osacky/flank/gradle/FladleConfig.kt

```diff
@@ -256,4 +256,44 @@ interface FladleConfig {
    */
   @get:Input
   val fullJunitResult: Property<Boolean>
+
+  /**
+   * A list of up to 100 additional APKs to install, in addition to those being directly tested.
```

Comment:
what's the different between this and `additionalTestApks`?

Reply:
https://cloud.google.com/sdk/gcloud/reference/alpha/firebase/test/android/run#--additional-apks

`additionalTestApks` is flank specific option - test apk pairs, meaning app.apk/test.apks to be tested
`additionalApks` is gcloud's option - apps to be installed but not tested (example apk to test is depending on another app)